Library/Formula/bugzilla.rb

```diff
@@ -0,0 +1,22 @@
+class Bugzilla < Formula
+  desc "Bugzilla is server software designed to help you manage software development."
```

Comment:
The name will be attached to the description so this can just be 
```ruby
desc "Server software designed to help you manage software development."
```